README.md

```diff
@@ -46,23 +47,24 @@ A response example is provided below.
 ```
 
 ##Authors by UUID
-`GET /transformers/authors/{uuid}` returns author data of the given uuid. 
+`GET /transformers/author-memberships/{uuid}` returns author membership data of the given uuid. 
 A response example is provided below.
 
 ```
 {
-  "uuid": "daf5fed2-013c-468d-85c4-aee709b8aa53",
-  "identifiers": [
+  "uuid": "e6e8b0e4-4833-11e6-beb8-9e71128cae77",
+  "prefLabel": "Chief Economics Commentator",
+  "personUuid": "6f53299a-799d-49ae-ae9d-7e1f298daef7",
+  "organisationUuid": "dac01f07-4b6d-3615-8532-a56752cc7e5f",
+  "alternativeIdentifiers": {
+    "uuids": [
+      "e6e8b0e4-4833-11e6-beb8-9e71128cae77"
+    ]
+  },
+  "membershipRoles": [
     {
-      "authority": "http://api.ft.com/system/FT-TME",
-      "identifierValue": "Q0ItMDAwMDkwMA==-QXV0auiycw=="
+      "roleuuid": "7ef75a6a-b6bf-4eb7-a1da-03e0acabef1b"
```

Comment:
It works still however the return JSON is wrong but it still writes successfully:

{
  "uuid": "2b92cac6-dbbf-4d74-9df7-f548bf89a333",
  "prefLabel": "Son in the and Sons business",
  "personUuid": "d4b051c7-5f4a-4188-9a20-826d3b17c0ba",
  "organisationUuid": "9441e9aa-b77b-b726-2191-6ef63f6c2a45",
  "inceptionDate": "2004-01-01T00:00:00.000Z",
  "terminationDate": "2005-01-01T00:00:00.000Z",
  "alternativeIdentifiers": {
    "uuids": []
  },
  "membershipRoles": [
    {
      "roleuuid": "c9c9cae0-afe6-30ca-b7e6-fe5cd9dccb60",
      "inceptionDate": "2004-01-01T00:00:00.000Z"
    },
    {
      "roleuuid": "eaa6f59e-b24c-3d36-8b79-062381f828e0",
      "inceptionDate": "2004-01-01T00:00:00.000Z",
      "terminationDate": "2005-01-01T00:00:00.000Z"
    },
    {
      "roleuuid": "73b9c823-aa25-33b7-abe8-1e6d830ded55",
      "inceptionDate": "2004-01-01T00:00:00.000Z"
    }
  ]
}